### PR TITLE
IcecreamApps.IcecreamEbookReader version 6.25

### PR DESCRIPTION
**Pull request by CanePlayz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/99218)

### CODE REVIEW

**Comment by stephengillie** [comment]
Hash verification failed in a sandbox:
> 2023-03-13 10:27:49.325 [CLI ] Package hash verification failed. SHA256 in manifest [4664b6c16f5ecdc45786b3684708c804d92ce1a4df11376a8a33b560894af81a] does not match download [2050bc4d7b56e2e2a7b40bc1134aeed5eb530af654ac8934e52bfe8963c36151]

File: manifests/i/IcecreamApps/IcecreamEbookReader/6.25/IcecreamApps.IcecreamEbookReader.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: IcecreamApps.IcecreamEbookReader
+PackageVersion: 6.25
+Installers:
+- Architecture: neutral
+  InstallerType: inno
+  InstallerUrl: https://icecreamapps.com/download/ebook_reader_setup.exe
+  InstallerSha256: 4664B6C16F5ECDC45786B3684708C804D92CE1A4DF11376A8A33B560894AF81A
```
Comment (stephengillie) [inline manifests/i/IcecreamApps/IcecreamEbookReader/6.25/IcecreamApps.IcecreamEbookReader.installer.yaml:10]:
```suggestion
  InstallerSha256: 2050BC4D7B56E2E2A7B40BC1134AEED5EB530AF654AC8934E52BFE8963C36151
```

**Comment by CanePlayz** [comment]
Alright, thanks, that's because there's a new version. I will update the PR.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**